src/Uno.UI.Runtime.Skia.Wpf/Islands/UnoXamlHostBase.cs

```diff
@@ -0,0 +1,361 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Uno.UI.Runtime.Skia.Wpf.Hosting;
+using Uno.UI.Skia.Platform;
+using Uno.UI.XamlHost;
+using WUX = Windows.UI.Xaml;
+
+namespace Uno.UI.XamlHost.Skia.Wpf
+{
+	/// <summary>
+	/// UnoXamlHost control hosts UWP XAML content inside the Windows Presentation Foundation
+	/// </summary>
+	abstract partial class UnoXamlHostBase
```

Comment:
Please keep all the class access modifiers in the same "main" file (and use only `partial class` in partial files)

```suggestion
	public abstract partial class UnoXamlHostBase
```